tests/reference_scores/reference_task_scores.py

```diff
@@ -94,16 +94,6 @@
             "pqem": 0.6051987767584098,
             "pqem_stderr": 0.008549304887647408,
         },
-        "helm|hellaswag|5|0": {
```

Comment:
are those not valid anymore ?

Reply:
Nope, removed the way few shot were managed for HELM (was only used in this task) so these scores are no longer correct